src/edu/csus/ecs/pc2/shadow/RemoteEventFeedMonitor.java

```diff
@@ -464,20 +464,19 @@ public void run() {
                                             } catch (Exception e) {
                                                 // TODO design error handling reporting
                                                 if (Utilities.isDebugMode()) {
-                                                    System.err.println("Exception submitting run for: " + event);
+                                                    System.err.println("Exception submitting run for event: " + event + ": " + e);
                                                 }
-                                                log.log(Level.WARNING, "Exception submitting run for: \\" + event);
-                                                
+                                                log.log(Level.WARNING, "Exception submitting run for event: " + event + ": ", e);
```

Comment:
*** Note *** There are 3 comments about log messages, instead of adding a comment for every log change, a
single comment per "correction" has been added.

Reply:
I'm not sure what this comment is intended to request (if anything).